workers.go

```diff
@@ -369,23 +371,23 @@ func (n *Node) processReqStoreEvents(eventsIn *events.EventList) (*events.EventL
 
 			// Store request data.
 			if err := n.modules.RequestStore.PutRequest(storeEvent.RequestRef, storeEvent.Data); err != nil {
-				return nil, fmt.Errorf("cannot store request (c%dr%d) data: %w",
+				return nil, fmt.Errorf("cannot store request (c%sr%d) data: %w",
 					storeEvent.RequestRef.ClientId,
 					storeEvent.RequestRef.ReqNo,
 					err)
 			}
 
 			// Mark request as authenticated.
 			if err := n.modules.RequestStore.SetAuthenticated(storeEvent.RequestRef); err != nil {
-				return nil, fmt.Errorf("cannot mark request (c%dr%d) as authenticated: %w",
+				return nil, fmt.Errorf("cannot mark request (c%sr%d) as authenticated: %w",
 					storeEvent.RequestRef.ClientId,
 					storeEvent.RequestRef.ReqNo,
 					err)
 			}
 
 			// Store request authenticator.
 			if err := n.modules.RequestStore.PutAuthenticator(storeEvent.RequestRef, storeEvent.Authenticator); err != nil {
-				return nil, fmt.Errorf("cannot store authenticator (c%dr%d) of request: %w",
+				return nil, fmt.Errorf("cannot store authenticator (c%sr%d) of request: %w",
```

Comment:
Let's use `%v` so we don't need to change this again in the unlikely case we use a different type for IDs in the future (or in a variant of Mir)